application/app/Traits/FuzzySearchable.php

```diff
@@ -0,0 +1,10 @@
+<?php namespace App\Traits;
+
+trait FuzzySearchable
+{
+    // あいまい検索
+    public function scopeFuzzySearch($column_name, $value)
+    {
+        return $this->where($column_name, 'like', '%' . $value . '%');
+    }
```

Comment:
スコープの使い方おかしい、ドキュメント読んだ？
第1引数は必ずBuilderになる。
あとタイプヒンティング使うこと。

```suggestion
    use Illuminate\Database\Eloquent\Builder;
    
    public function scopeFuzzySearch(Builder $query, string $column_name, string $value)
    {
        return $query->where($column_name, 'like', '%' . $value . '%');
    }
```